vnext/Desktop.ABITests/ActivationFactory.cpp

```diff
@@ -0,0 +1,71 @@
+
+#include <Windows.h>
+#include <winrt/base.h>
+
+using namespace winrt;
+
+extern "C"
+{
+	int32_t WINRT_CALL OS_RoGetActivationFactory(void* classId, guid const& iid, void** factory) noexcept;
+}
+
+#ifdef _M_IX86
+#pragma comment(linker, "/alternatename:_OS_RoGetActivationFactory@12=_RoGetActivationFactory@12")
+#else
+#pragma comment(linker, "/alternatename:OS_RoGetActivationFactory=RoGetActivationFactory")
+#endif
+
+bool starts_with(std::wstring_view value, std::wstring_view match) noexcept
+{
+	return 0 == value.compare(0, match.size(), match);
+}
+
+int32_t WINRT_CALL WINRT_RoGetActivationFactory(void* classId, guid const& iid, void** factory) noexcept
+{
+	*factory = nullptr;
+	std::wstring_view name{ WINRT_WindowsGetStringRawBuffer(classId, nullptr) };
+	HMODULE library{ nullptr };
+
+	if (starts_with(name, L"facebook.react."))
+	{
+		library = LoadLibraryExW(RNDLLPATH, NULL, LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR | LOAD_LIBRARY_SEARCH_DEFAULT_DIRS);
+		auto le = GetLastError();
+		le = le;
```

Comment:
Needed?

Reply:
Removed.

---
In reply to: [298782488](https://github.com/microsoft/react-native-windows/pull/2656#discussion_r298782488) [](ancestors = 298782488)